sdk/providerhub/Microsoft.Azure.Management.ProviderHub/src/Microsoft.Azure.Management.ProviderHub.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.ProviderHub</PackageId>
+    <Description>Microsoft Azure Management Library for ProviderHub operations.</Description>
+    <Version>1.0.0</Version>
```

Comment:
Your fist version should likely be a preview.  version 1.0.0-beta.1  would be the correct version number for your first preview